openshift-examples/keycloak.yaml

```diff
@@ -60,40 +60,34 @@ objects:
         spec:
           containers:
             - env:
-                - name: KEYCLOAK_USER
-                  value: '${KEYCLOAK_USER}'
-                - name: KEYCLOAK_PASSWORD
-                  value: '${KEYCLOAK_PASSWORD}'
-                - name: DB_VENDOR
-                  value: '${DB_VENDOR}'
```

Comment:
Also, how is the DB supposed to be configured now? Should we add some params to the template?

Before, it started OOTB thanks to H2. This is not the case, right?